pkg/grpc/api.go

```diff
@@ -35,6 +38,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var log = logger.NewLogger("dapr.api-grpc")
```

Comment:
This is not needed. this file already uses `apiServerLogger`.

Reply:
fixed